sass/includes/_record-links.scss

```diff
@@ -0,0 +1,92 @@
+.record-links {
+    &__wrapper {
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+        padding-left:10%;
+        padding-right:10%;
+
+        @media (max-width: $screen__md) {
+            padding-left:5%;
+            padding-right:5%;
+        }
+    }
+
+    &__list {
+        list-style: none;
+        margin-bottom: 0px;
+        width: 100%;
+        padding-left: 0;
+        border-top: 0.3rem solid #ffcc00;
+    }
+
+    &__list a {
+        text-decoration: none;
+        color: inherit;
+    }
+    
+    &__list a:hover &__item,
+    &__list a:focus &__item{
+        outline: 0.312rem solid $color__dark;
+    }
+
+    &__item {
+        display: flex;
+        align-items: center;
+        border: 1px solid #D9D9D6;
+        padding-top: 2rem;
+        padding-bottom: 2rem;
+        width: 100%;
+
+        @media (max-width: $screen__lg) {
```

Comment:
As I mentioned above - lets swap out media queries for min-widths and move/swap the relevant styles as needed